src/utils/setup.ts

```diff
@@ -0,0 +1,44 @@
+import * as shell from 'shelljs';
```

Comment:
Ok, to do this we need to have public version of the blog template. Can you create that or empty repo to upload? Thanks!

Reply:
> 

I would put it directly into code so you are just writing out the file. No external source.

Reply:
Don't understand. Shelljs is for run commands like ```git init``` or ```git clone .....```

Reply:
would not use git. Just embed writing a file in the code such as:

```typescript

fs.writeLine(`-----`);
fs.writeLine(`Author: foo`);
```

write the file inside the code.

Reply:
Ok, including package.json file, right? we can have a template with that and just we need to write the file. Ok, perfect, not a problem. I will update.

Reply:
I dont think we need package.json file as it should only be the init for the blog to be generated. Just what is in blog_example correct?

Reply:
I try to add package.json because the next steps we can add writer as a devDependency to run watch command to generate the files when a new file change.